src/test/java/com/google/api/codegen/testdata/library_pkg.yaml

```diff
@@ -6,6 +6,9 @@ author: Google, Inc.
 email: googleapis-packages@google.com
 homepage: https://github.com/googleapis/googleapis
 license: Apache-2.0
+proto_gen_pkg_deps:
```

Comment:
Changed to `proto_deps` (and in googleapis/artman).

(I originally used `proto_gen_pkg_deps` because that was already the name of the field in the artman config.)